webapps/frontend/app/viewmodels/site/admin/support.js

```diff
@@ -44,13 +52,14 @@ define([
 
         // Handles
         self.supportMetadataHandle = {};
-        self.supportSettingsHandle = undefined;
+        self.supportSettings = undefined;
```

Comment:
Renaming might cause confusion. This is a handle to avoid multiple concurrent requests.